src/styles/globals.css

```diff
@@ -29,6 +29,7 @@ body {
   scroll-padding-top: 6rem;
 }
 
+
```

Comment:
as you are doing other actions can you also remove this extra line, not needed

Reply:
Okay, will do!